src/components/StudyList/StudyCard.js

```diff
@@ -1,24 +1,138 @@
-import React from 'react';
+import React, {useState} from 'react';
 import PropTypes from 'prop-types';
-import {Link} from 'react-router-dom';
-import {Card} from 'semantic-ui-react';
+import {Link, withRouter} from 'react-router-dom';
+import {Card, Icon, Label, Button, Popup} from 'semantic-ui-react';
 import FileCounts from '../StudyInfo/FileCounts';
-
+import CavaticaCounts from '../StudyInfo/CavaticaCounts';
+import {trackedStudyFields} from '../../common/notificationUtils';
 /**
  * Displays each study with its kfId, name(shortName), and modifiedAt
  */
-
-const StudyCard = ({title, body, files, lastUpdate}) => {
+const StudyCard = ({
+  title,
+  body,
+  files,
+  projects,
+  history,
+  lastUpdate,
+  missingValue,
+  missingProject,
+  requiredFileChanges,
+}) => {
+  const projectsCounts = projects && projects.length > 0 ? projects.length : 0;
+  const needsAttention =
+    missingValue > 0 ||
+    missingProject > 0 ||
+    requiredFileChanges > 0 ||
+    projectsCounts < 1;
+  const [showDetail, setShowDetail] = useState(false);
   return (
-    <Card as={Link} to={`/study/${title}/documents`}>
-      <Card.Content>
-        <Card.Header>{title}</Card.Header>
-
-        <Card.Description>{body}</Card.Description>
-      </Card.Content>
-      <Card.Content extra>
-        <FileCounts files={files} />
+    <Card
+      to={`/study/${title}/documents`}
```

Comment:
🤔 maybe we could standardize all of our route strings across the app by putting them into [template literals](
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals) in `/common` somewhere? This way as the app grows it will make updating routes easier. Might be a good case for a new PR to do so though 

@dankolbman thoughts?